ql/src/java/org/apache/hadoop/hive/ql/io/sarg/ConvertAstToSearchArg.java

```diff
@@ -591,4 +608,22 @@ public static String sargToKryo(SearchArgument sarg) {
     }
   }
 
+  public static class Result {
```

Comment:
please add `final`

Reply:
note, we should create a refactor jira for that class

Reply:
Added final. Done.